airbyte-integrations/connectors/source-mailchimp/integration_tests/invalid_config.json

```diff
@@ -1,4 +1,6 @@
 {
-  "apikey": "api-key-awesome",
-  "username": "fakemaster@silly.com"
+  "credentials": {
```

Comment:
Please leave this config without changes and add new ouath_invalid_config.json

We need to check if connector still can work with old config

Reply:
Leave old `invalid_config.json` and add with new credentials format `invalid_config_apikey.json.`